src/app/main/component/events/services/events.service.ts

```diff
@@ -28,6 +28,10 @@ export class EventsService implements OnDestroy {
     return this.http.get(`${this.backEnd}events/myEvents?page=${page}&size=${quantity}`);
   }
 
+  public getCreatedEvents(page: number, quantity: number): Observable<any> {
```

Comment:
Why any? Can we create an interface for response?

Reply:
Yes, I added interface for response it in new commit.